programs/bpf/rust/ro_modify/src/lib.rs

```diff
@@ -143,10 +135,10 @@ fn process_instruction(
                 check!(
                     0,
                     sol_invoke_signed_c(
-                        &instruction as *const _,
-                        READONLY_ACCOUNTS.as_ptr(),
+                        &instruction as *const _ as *const _,
```

Comment:
duplicate cast?

Reply:
It's not a duplicate: the first casts &SolInstruction to *const Instruction, the second casts that to *const u8